File: QvAbu.Web/src/app/models/questions/assignment-question.ts
import { Question, QuestionType } from './question';
import { AssignmentOption } from './assignment-option';
import { AssignmentAnswer } from './assignment-answer';
import { Guid } from '../guid';

export class AssignmentQuestion extends Question {
  options: AssignmentOption[];
  answers: AssignmentAnswer[];

  constructor(id: Guid) {
    super();
    this.id = id;

    this.type = QuestionType.assignmentQuestion;
  }
}

File: QvAbu.Data/Models/Questions/AssignmentQuestion.cs
using System.Collections.Generic;

namespace QvAbu.Data.Models.Questions
{
    public class AssignmentQuestion : Question
    {
        #region Properties

        public List<AssignmentOption> Options { get; set; }
        public List<AssignmentAnswer> Answers { get; set; }

        #endregion
    }
}
